src/widget/scroll.rs

```diff
@@ -139,6 +147,49 @@ impl<T: Data, W: Widget<T>> Scroll<T, W> {
         self.scroll_offset
     }
 
+    fn calc_scrollbar_screen_edges(&self, viewport: &Rect) -> Rect {
+        return Rect::new(
+            self.scroll_offset.x + SCROLL_BAR_PAD,
+            self.scroll_offset.y + SCROLL_BAR_PAD,
+            self.scroll_offset.x - SCROLL_BAR_PAD + viewport.width(),
+            self.scroll_offset.y - SCROLL_BAR_PAD + viewport.height(),
+        );
```

Comment:
there is now an [`Insets`] type in kurbo that was intended to be useful in situations like this. (I'm not sure it would be useful here, but it might be worth checking out?)

[`Insets`]: https://docs.rs/kurbo/0.5.4/kurbo/struct.Insets.html

Reply:
This code was also inherited, personally I don't like it but I didn't want to modify the existing way of doing things to much